sp800-63c/sec5_federation.md

```diff
@@ -1,65 +1,65 @@
 <div class="breaker"></div>
-<a name="sec4"></a>
+<a name="federation"></a>
 
-## 4. Federation
+## 5. Federation
 
 *This section is normative.*
 
-In a federation protocol, a three-party relationship is formed between the subscriber, the IdP, and the RP as shown in [Figure 4-1](#63cSec4-Figure1). Depending on the specifics of the protocol, different information passes among the participants at different times. The subscriber communicates with both the IdP and the RP, usually through a browser. The RP and the IdP communicate with each other in two ways:
+In a federation protocol, a three-party relationship is formed between the subscriber, the IdP, and the RP as shown in [Figure 5-1](#63cSec5-Figure1). Depending on the specifics of the protocol, different information passes among the participants at different times. The subscriber communicates with both the IdP and the RP, usually through a browser. The RP and the IdP communicate with each other in two ways:
 
  - The *front channel*, through redirects involving the subscriber; or
  - The *back channel*, through a direct connection between the RP and IdP, not involving the subscriber.
 
-<a name="63cSec4-Figure1"></a>
+<a name="63cSec5-Figure1"></a>
 
 <div class="text-center" markdown="1">
 ![Figure 1: Federation](sp800-63c/media/federation.png)
 
-**Figure 4-1. Federation**
+**Figure 5-1. Federation**
 
 </div>
 
-The subscriber authenticates to the IdP as described in [[SP 800-63B]](sp800-63b.html), and then the result of that authentication event is asserted to the RP across the network. The IdP can also make attribute statements about the subscriber as part of this process. These attributes and authentication event information are carried to the RP through the use of an assertion, described in [Section 5](#sec5).
+The subscriber authenticates to the IdP as described in [[SP 800-63B]](sp800-63b.html), and then the result of that authentication event is asserted to the RP across the network. The IdP can also make attribute statements about the subscriber as part of this process. These attributes and authentication event information are carried to the RP through the use of an assertion, described in [Section 6](#assertions).
```

Comment:
Good thought -- what we really want to say here is that the authentication process from the user to the IdP is covered entirely by B. Should be a simple matter of mapping terminology back. (We already map it forward in the intro)